.github/workflows/kserve_kind_test.yaml

```diff
@@ -0,0 +1,44 @@
+name: Build & Apply KServe manifests in KinD
+on:
+  pull_request:
+    paths:
+      - contrib/kserve/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install KinD
+      run: ./testing/gh-actions/install_kind.sh
+
+    - name: Create KinD Cluster
+      run: kind create cluster --config testing/gh-actions/kind-1-22.yaml
+
+    - name: Install kustomize
+      run: ./testing/gh-actions/install_kustomize.sh
+
+    - name: Install Istio
+      run: |
+        cd common/istio-1-11
+        kustomize build istio-crds/base | kubectl apply -f -
+        kustomize build istio-namespace/base | kubectl apply -f -
+        kustomize build istio-install/base | kubectl apply -f -
+
+    - name: Install cert-manager
+      run: |
+        cd common/cert-manager
```

Comment:
Since we use this in a couple of places let's also create script

Reply:
I added the script.